### ISSUE TITLE
features split in class tri_GRU

### ISSUE THREAD

**Issue by tdcqhxr** [akshat3011/punchline-detection-hierarchical-fusion#1]
Thank you for the code to make it easier for model understanding.
I have a question about the features in the class tri_GRU: 
the punchline features and context  features are  got by     
    x_p = x[:,1,:]
    x_c = x[:,1:6,:]
but the features got from dataset is
combined = torch.cat([x_c, x_p], dim=1)
I did not understand the way of the split, could you please explain it?
